src/Longitude.cpp

```diff
@@ -2,6 +2,12 @@
 #include "Utility.hpp"
 #include "Longitude.hpp"
 
+#if INDI_SUPPORT == 1
+    #define LONGITUDE_OFFSET 0L
```

Comment:
Could you also add in
```C++
#pragma message("Note: INDI offset used, firmware will not function correctly with ASCOM!")
```
? Just so that we can have traceability if a user is having problems.